Changelog.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 ## V4.0
 
+### Added

```

Comment:
Don't put this in 4.0. That was already released. Add it to the Unreleased entry I committed.